warehouse/accounts/views.py

```diff
@@ -53,6 +53,7 @@ def failed_logins(exc, request):
 @view_config(
     route_name="accounts.profile",
     renderer="accounts/profile.html",
+    uses_session=True,
```

Comment:
thanks for catching this @di.
sounds good @dstufft.  Would you make a separate PR for this, or include it in this one? (FWIW, I'm totally happy for you to edit this PR directly)